app/templates/components/paper-toast.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if hasBlock}}
+  {{yield}}
+
```

Comment:
I personally think they make the file more readable by separating the parts into chunks. If that is required by the style guide I will change it though.